app/views/users/_form.html.erb

```diff
@@ -1,6 +1,7 @@
 
 <%= form_for(@resource) do |f| %>
   <%= f.text_field :user_name %>
+  <%= f.check_box :mentor, value: @resource.mentor %>
```

Comment:
Nope, not needed. Good catch @sampritipanda  I have to learn that Godmin is doing a lot of stuff for us...